testprojects/tests/java/org/pantsbuild/testproject/resourcejar/ResourceJar.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.testproject.resourcejar;
+
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * Demonstrate that instead of a directory of resources on the classpath we put a zip of the
+ * resources on the classpath.
+ */
+public class ResourceJar {
+
+  @Test
+  public void testResourceJar() throws Exception {
+    // Test part 1: make sure the resource jar is named expectedly and does exist.
```

Comment:
Rather than matching the jar name, should just test that a particular resource you have defined is located in a jar. The name isn't relevant.